### ISSUE TITLE
Consider moving configuration into env files or similar

### ISSUE THREAD

**Issue by bmann** [aaroncox/steempress#2]
@aaroncox just getting into running and configuring.

prod.php and other files are checking in git code, which makes it awkward to change those files just for my instance. Would be better if those were reading from an .env file that isn't checked into git.

I think I'm going to run a separate branch for now, which is what I'll set up to deploy to Heroku, and keep my master clean and tracking your stuff.

**Comment by aaroncox** [comment]
Yeah, those files won't be in the repository eventually. Generally what I'll do is add the `config/*.php` files to the .gitignore file, and then create templates/examples in the folder that you can copy into place as your configuration files.

I'll look into what options silex provides, and see what would be best for the docker/heroku combo configuration we've got running here now.
